test/tap/tests/test_admin-tls_version-t.cpp

```diff
@@ -0,0 +1,292 @@
+/**
+ * @file test_simple_ssl_con-t.cpp
+ * @brief  *   Simple test for testing SSL connections to ProxySQL.
+ */
+
+#include <cstring>
+#include <cmath>
+#include <vector>
+#include <string>
+
+#include <mysql.h>
+#include <mysql/errmsg.h>
+
+// required for utility functions
+#include <gen_utils.h>
+
+#include "proxysql_utils.h"
+#include "tap.h"
+#include "command_line.h"
+#include "utils.h"
+
+/**
+ * @brief Copy of the supported 'tls_versions' by ProxySQL.
+ */
+const std::array<const char*, 4> valid_tls_versions {
```

Comment:
valid versions do not include any value with different cases, like `tlsv1.3` ot `tLsV1.1`